notes-to-self/proxy-benchmarks.py

```diff
@@ -15,53 +16,67 @@ def __getattr__(self, name):
             return getattr(self._wrapped, name)
         raise AttributeError(name)
 
+
 ################################################################
 
+
 class Proxy2:
     strategy = "generated methods (getattr + closure)"
     works_for = "methods"
 
     def __init__(self, wrapped):
         self._wrapped = wrapped
 
+
 def add_wrapper(cls, method):
     def wrapper(self, *args, **kwargs):
         return getattr(self._wrapped, method)(*args, **kwargs)
+
     setattr(cls, method, wrapper)
 
+
 for method in methods:
     add_wrapper(Proxy2, method)
 
 ################################################################
 
+
 class Proxy3:
     strategy = "generated methods (exec)"
     works_for = "methods"
 
     def __init__(self, wrapped):
         self._wrapped = wrapped
 
+
 def add_wrapper(cls, method):
-    code = textwrap.dedent("""
+    code = textwrap.dedent(
+        """
         def wrapper(self, *args, **kwargs):
             return self._wrapped.{}(*args, **kwargs)
-    """.format(method))
+    """.format(
+            method
+        )
+    )
```

Comment:
golly this is ugly, but not worth doing anything about

Reply:
Yea definitely not very pretty